modules/imaging_browser/jsx/ImagingBrowserDDT.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by dblader on 7/25/17.
+ */
+/* global DynamicDataTable */
+class ImagingBrowserDDT extends DynamicDataTable {
```

Comment:
@driusan Would you suggest then having this feature placed in main StaticDataTable class with a prop that is passed to flag this behaviour to run?